package.json

```diff
@@ -37,13 +37,13 @@
         "command": "vscode-manim.previewManimCell",
         "key": "ctrl+' ctrl+e",
         "mac": "cmd+' cmd+e",
-        "when": "editorTextFocus"
+        "when": "(editorTextFocus || terminalFocus) && editorLangId == 'python'"
```

Comment:
Sure, let's do it. You don't think it is confusing as to "which line is active" in the og python file when the terminal is active?

Reply:
well, if a user is using a shortcut when the terminal is active, this user probably remembers the line